submissions/Temu4/html_css_popup/index.html

```diff
@@ -0,0 +1,169 @@
+                <img src="img/apps-dairy.png" alt="Calendar" />
+                <div class="app-item-label">Calendar</div>
+              </td>
+            </tr>
+            <tr>
+              <td tabindex="13">
+                <img src="img/apps-bussines.png" alt="Business" />
+                <div class="app-item-label">My Business</div>
+              </td>
+            </tr>
+          </table>
+
+          <!-- Button More -->
+          <button class="apps-more-btn" tabindex="14">More</button>
+          <!-- Container with more apps list -->
+          <div class="apps-more-list">
+            <div class="divider"></div>
+            <table class="table-apps-more">
+              <tr>
+                <td tabindex="15">
+                  <img src="img/apps-blogger.png" alt="Blogger" />
+                  <div class="app-item-label">Blogger</div>
+                </td>
+                <td tabindex="16">
+                  <img src="img/apps-photos.png" alt="Photos" />
+                  <div class="app-item-label">Photos</div>
+                </td>
+                <td tabindex="17">
+                  <img src="img/apps-map.png" alt="Maps" />
+                  <div class="app-item-label">Maps</div>
+                </td>
+              </tr>
+              <tr>
+                <td tabindex="17">
+                  <img src="img/apps-acc.png" alt="Account" />
+                  <div class="app-item-label">Account</div>
+                </td>
+                <td tabindex="18">
+                  <img src="img/apps-contact.png" alt="Contact" />
+                  <div class="app-item-label">Contact</div>
+                </td>
+                <td tabindex="19">
+                  <img src="img/apps-massager.png" alt="Massanger" />
+                  <div class="app-item-label">Massanger</div>
+                </td>
+              </tr>
+              <tr>
+                <td tabindex="20">
+                  <img src="img/apps-keep.png" alt="Keep" />
+                  <div class="app-item-label">Keep</div>
+                </td>
+                <td tabindex="21">
+                  <img src="img/apps-jumboard.png" alt="Jambord" />
+                  <div class="app-item-label">Jambord</div>
+                </td>
+                <td tabindex="22">
+                  <img src="img/apps-classroom.png" alt="Classroom" />
+                  <div class="app-item-label">Classroom</div>
+                </td>
+              </tr>
+              <tr>
+                <td tabindex="23">
+                  <img src="img/apps-earth.png" alt="Planet" />
+                  <div class="app-item-label">Planet</div>
+                </td>
+                <td tabindex="24">
+                  <img src="img/apps-collection.png " alt="Collections" />
+                  <div class="app-item-label">Collections</div>
+                </td>
+              </tr>
+            </table>
+
+            <div class="apps-even-more-link">
```

Comment:
`class="apps-even-more-link"`
It's BEM ?

Reply:
> It's BEM ?

 Sorry, could you explain?

Reply:
[link](https://en.bem.info/) Good methodology for naming class.
Recomend for read =)